libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -280,10 +280,7 @@ def deploy_cluster(self, node_obj: LogicalNode, remote_code_path: str) -> tuple:
         return : True/False and resp
         """
         LOGGER.info("Deploy Cortx cloud")
-        export_cmd = common_cmd.LINUX_EXPORT.format(self.deploy_cfg["deploy_ha_timeout_key"],
-                                                    str(self.deploy_cfg["deploy_ha_timeout_val"])
-                                                    + "s")
-        cmd = export_cmd + " && " + common_cmd.DEPLOY_CLUSTER_CMD.format(
+        cmd = common_cmd.DEPLOY_CLUSTER_CMD.format(
```

Comment:
Use spring template wherever you are using format.

Reply:
used string Template